normandy/recipes/tests/test_serializers.py

```diff
@@ -15,6 +15,7 @@ def test_it_works(self, rf):
         assert serializer.data == {
             'name': recipe.name,
             'id': recipe.id,
+            'last_updated': recipe.last_updated.replace(tzinfo=None).isoformat() + 'Z',
```

Comment:
I like `Whatever()` better as well. Less ugly and maybe it's not something that's super important to test the exact value of, just that it exists.